### PR TITLE
CryptoKings - React Capstone Project

### PR DESCRIPTION
**Pull request by PabloBona**
Description:
In this React capstone project, I developed a mobile web application called CryptoKings. The application allows users to explore a list of cryptocurrency coins and view their details. Users can filter the coins based on specific parameters such as name. The application fetches real-time data from an open cryptocurrency API.

**How it Works:**
The CryptoKings web app consists of two main sections:

**1. Home Page - Coin List:**
The Home Page displays a list of cryptocurrency coins along with their respective numeric values, which are fetched from the API. Users can filter the coins based on the parameter name. Clicking on a coin item will navigate the user to the details page for that coin.

**2. Coin Details Page:**
On the Coin Details Page, users can view more detailed information about a specific cryptocurrency coin. This page fetches detailed data about the selected coin from the API. The user can then click the "Back" button (<) to return to the Home Page and explore other coins.

**Built With**
The CryptoKings web application is built using the following tech stack:

- React
- Redux
- React Router
- React Testing Library (for integration tests)
- Jest (for unit tests)
- API for real-time cryptocurrency data

**Design Guidelines:**
The CryptoKings web application follows the design guidelines outlined in the Figma design template by Nelson Sakwa. The design includes a selected main color scheme, typography, and layout composition. Credits: Original design idea by Nelson Sakwa on Behance. The design is used under the Creative Commons license, and appropriate credit is given to the author.

**Testing:**
We have included both unit tests for pure functions in the Redux code and integration tests using React Testing Library. The tests ensure the application's functionality and behavior are thoroughly checked.

**Technical Requirements:**
- The project is a single-page application (SPA) built with React and Redux.
- The data retrieved from the API is stored in the Redux store.
- Filtering of the API data is done using a Filter stateless component.
- Each page (main page and pages for each item) has a unique route within the SPA.
- The project is deployed and accessible online via [Link to your deployed app].

**Live Demo:**
You can access the live demo of the CryptoKings web application by visiting [[Link to live demo](https://cryptoskings.netlify.app/)].

**Presentation Video:**
A presentation video showcasing the functionality and features of the CryptoKings web application can be found at [[Link to presentation video](https://drive.google.com/file/d/1ebLDgW4fpoal4sebBb46jKsV17lJeLkh/view)].

### CODE REVIEW

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hi @PabloBona  

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

# Invalid Code Review Request 🔴 

You have submitted a project, that has failing linters

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by PabloBona** [comment]
@Shedrack-Sunday
Hello! The error that occurred was due to the deployment on Netlify, not a linter error!
Warm regards.

File: README.md [hunk 1]
```diff
@@ -78,7 +80,13 @@ Credits: Original design idea by Nelson Sakwa on Behance. The design is used und
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 ### 🚀 Live Demo <a name="live-demo"></a>
-<a href="https://crypto-kings.netlify.app/">Live Demo</a>
+<a href="https://cryptoskings.netlify.app/">Live Demo</a>
+
```
Comment (bobb-Rob) [inline README.md:84]: - [ ] Based on the project requirement, you should the acknowledgment section that gives appropriate credit to  [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

**Review by bobb-Rob — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️
Hi @PabloBona , 

Good job so far! 👏👏👏 
There are some issues that you still need to work on to go to the next project but you are almost there!  😃 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Highlights
✅ All linters checks are ok
✅ The correct workflow is used. 
✅ Very descriptive Pull Request(PR)
✅ Your work is well documented.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (PabloBona) [reply README.md:84]: @bobb-Rob 
image
Is here !! But i m happy if that is the only issue in my project !!!
Warm regards. !!!

**Review by KanzaTahreem — APPROVED** [review]
# Approved 🙌

Hi @PabloBona 

Good job implementing the suggested changes💪 **_Your project is COMPLETE! There is nothing else to say other than... it's time to merge it! 🚀 Congratulations!_**



## Highlights 🎯
There are some excellent points that you have made perfectly. Keep up the great work! 👏

-  ✅ Styling is looking great, perfect!
-  ✅ Acknowledgment section is added 
-  ✅ Your README looks good, well done!

## Optional suggestions ❕
- [OPTIONAL] I noticed that there are a few other previously merged PRs in your repository from feature branches to the dev branch. Please make sure that any PR following the GitFlow workflow should only be merged once it has been approved. This helps maintain the stability and integrity of the codebase. Please make sure to carefully go through the work and submission mode for you upcoming projects.✨📌

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**